### PR TITLE
2179  Add wait-for-confirmations flag

### PR DESCRIPTION
**Pull request by halink0803**

### CODE REVIEW

File: tradelogs/cmd/trade-logs-crawler/main.go [hunk 1]
```diff
@@ -269,7 +279,7 @@ func run(c *cli.Context) error {
 			if fErr != nil {
 				return fErr
 			}
-			toBlock = currentHeader.Number
+			toBlock = currentHeader.Number.Sub(currentHeader.Number, big.NewInt(blockConfirmation))
```
Comment (favadi) [inline tradelogs/cmd/trade-logs-crawler/main.go]: I can think of a scenario like that:

- from-block: 100
- to-block: 102

crawler will try to crawl from 100 --> 102 - 7, from-block < to-block --> exit. Could you double check?

Comment (halink0803) [reply tradelogs/cmd/trade-logs-crawler/main.go]: Yes, I already check it. The logic already implement here:
https://github.com/KyberNetwork/reserve-stats/blob/a60ba3754e42d41af4372e1fad11b890e3ca62c7/tradelogs/cmd/trade-logs-crawler/main.go#L286

Comment (favadi) [reply tradelogs/cmd/trade-logs-crawler/main.go]: but that means the program will returns an error and exit?

Comment (halink0803) [reply tradelogs/cmd/trade-logs-crawler/main.go]: @favadi I change the logic to sleep(delayTime) when latest block - block confirmations is smaller fromBlock.

**Pull request merged**